ui/intermittent-failures/Layout.jsx

```diff
@@ -36,7 +39,13 @@ const Layout = props => {
   }
   return (
     <Container fluid className="my-5 max-width-default">
-      <Navigation updateState={updateState} tree={tree} />
+      <Navigation
+        updateState={updateState}
+        tree={tree}
+        user={user}
+        setUser={setUser}
+        notify={notify}
+      />
```

Comment:
Since `user`, `setUser` and `notify` aren't actually being used in this component other than being passed into `Navigation` you can pass them via `{...this.props}`.